src/auth-service/routes/api.js

```diff
@@ -40,8 +40,8 @@ router.post("/forgotPassword", joinController.forgotPassword);
 router.get("/findUser", joinController.findUser);
 router.put("/", joinController.updateUser);
 router.delete("/:id", joinController.deleteUser);
-router.put("/defaults/:id", joinController.updateUserDefaults);
-router.get("/defaults/:id", joinController.getDefaults);
+router.put("/defaults", joinController.updateUserDefaults);
+router.get("/get/defaults", joinController.getDefaults);
```

Comment:
I think this should be  **/defaults**

Reply:
Yeah, true...there is a problem with routing module which causes errors if 2 routes of different HTTP verbs have the same paths. This was just a workaround the issue. 

It is kinda frustrating but let me continue investigating.

Reply:
> I think this should be **/defaults**

This has just been addressed in the most recent commits. PR description has also been updated accordingly.